requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==4.1
+Django==4.2.6
 mysqlclient==2.1.1
-django-testing-utils==0.5.0
+django-testing-utils
```

Comment:
DTU will support Dj4.2 in next version, see https://github.com/just-work/django-testing-utils/pull/62

I guess `==0.6.0` will be needed here.

Reply:
Yes, but no this tag yet.

Reply:
https://pypi.org/project/django-testing-utils/ released 0.6.0

Reply:
Version fixed